janitor/functions.py

```diff
@@ -8,7 +8,35 @@
 import re
 
 
-def clean_names(df):
+def _strip_underscores(df, strip_underscores=None):
+    """
+    Strip underscores from the beginning, end or both of the
+    of the DataFrames column names.
+
+    .. code-block:: python
+
+        df = _strip_underscores(df, strip_underscores='left')
+
+    :param df: The pandas DataFrame object.
+    :param strip_underscores: A str of either 'left', 'right' or 'both'.
```

Comment:
Please document the options in line 24, and what they're intended to do.

Reply:
Added better documentation of `strip_underscores`.